assets/src/scripts/_form-setup.js

```diff
@@ -1,10 +1,17 @@
-import { reviewUrl } from "./_data";
+import { reviewUrl, outputs } from "./_data";
 
 const formSetup = () => {
   const form = document.querySelector("#approveForm");
   const button = form.querySelector(`button[type="submit"]`);
   form.action = reviewUrl;
 
+  form.addEventListener("formdata", (ev) => {
+    // TODO: get from signal
+    for (const [output] of outputs) {
+      ev.formData.append("outputs", output);
+    }
+  });
```

Comment:
```suggestion
    [...outputs].forEach(([output]) => ev.formData.append("outputs", output));
```

I'm not a huge fan of `for ... of` loops, so you could change to a `forEach` iterator.

Reply:
Why the preference, OOI?

Reply:
These two links probably explain it better than I can:

- https://github.com/airbnb/javascript/issues/1271#issuecomment-283735174
- https://github.com/airbnb/javascript/issues/1271#issuecomment-283736133

Although the flame war in that thread suggests I may be wrong.